menu/cbs/menu_cbs_get_value.c

```diff
@@ -1388,7 +1388,7 @@ static void menu_action_setting_disp_set_label_menu_file_cheat(
       char *s2, size_t len2)
 {
    menu_action_setting_generic_disp_set_label(w, s, len,
-         path, "(CHEAT)", s2, len2);
+         path, NULL, s2, len2);
```

Comment:
Why are we trimming the filenames under CHEATS? Removing (CHEAT) makes a huge different in parsing filenames.